src/datadog/filter/regex.rs

```diff
@@ -4,6 +4,7 @@ use regex::Regex;
 ///
 /// # Panics
 /// Panics if an invalid wildcard regex is provided.
+#[allow(clippy::module_name_repetitions)]
 #[must_use]
 pub fn word_regex(to_match: &str) -> Regex {
```

Comment:
Same here, could just call this `word` and the below `wildcard` given the module is called `regex`.